perf/istio/external-traffic/values.yaml

```diff
@@ -0,0 +1,3 @@
+fortioImage: fortio/fortio:latest
+externalDestination: http://www.google.com
```

Comment:
Maybe hit istio.io ? Or the URL of the test cluster ? Pretty sure google.com will detect and start serving captchas or something like that.

Reply:
In `setup.sh` we find the ingress external IP so this will actually always be overriden. I think I'll need to remove this or at least comment it though so that is clear